src/utils/process.rs

```diff
@@ -702,53 +653,6 @@ pub mod tests {
         }
     }
 
-    #[test]
-    fn test_guess_git_blame_filename() {
-        use ProcessArgs::Args;
-
-        fn make_string_vec(args: &[&str]) -> Vec<String> {
-            args.iter().map(|&x| x.to_owned()).collect::<Vec<String>>()
-        }
-        let args = make_string_vec(&["git", "blame", "hello", "world.txt"]);
-        assert_eq!(guess_git_blame_filename(&args), Args("world.txt".into()));
-
-        let args = make_string_vec(&[
-            "git",
-            "blame",
-            "-s",
-            "-f",
-            "hello.txt",
-            "--date=2015",
-            "--date",
-            "now",
-        ]);
-        assert_eq!(guess_git_blame_filename(&args), Args("hello.txt".into()));
-
-        let args = make_string_vec(&["git", "blame", "-s", "-f", "--", "hello.txt"]);
-        assert_eq!(guess_git_blame_filename(&args), Args("hello.txt".into()));
-
-        let args = make_string_vec(&["git", "blame", "--", "--not.an.argument"]);
-        assert_eq!(
```

Comment:
Is this case still tested? And the failure cases, such as `git blame --help.txt`?

Reply:
I added more tests